### PR TITLE
Bugfix/broken responsiveness

### PR DESCRIPTION
**Pull request by anton-karlovskiy**

### CODE REVIEW

File: src/pages/app/issue/issue-requests.tsx [hunk 1]
```diff
@@ -1,18 +1,19 @@
-import React, { useState, ReactElement } from 'react';
```
Comment (nud3l) [inline src/pages/app/issue/issue-requests.tsx:1]: First line empty?

Comment (anton-karlovskiy) [reply src/pages/app/issue/issue-requests.tsx:1]: Actually, it should be from an ESLinting rule which I have not set.
This was set for me to check the code I touched.

**Review by nud3l — APPROVED** [review]

File: src/pages/app/issue/issue-requests.tsx (hunk 1 above)
Comment (nud3l) [reply src/pages/app/issue/issue-requests.tsx:1]: You want to update this in this PR?

Comment (anton-karlovskiy) [reply src/pages/app/issue/issue-requests.tsx:1]: I'm afraid not.
I'm going to work on another PR.

**Pull request merged**